front/src/services/__tests__/stringToTimeframeObject.tsx

```diff
@@ -31,14 +31,4 @@ describe("stringToTimeframeObject", () => {
             expect(difference).toEqual(604800000);
         },
     );
-    it(
```

Comment:
Why the removed test? 😂

Reply:
The types ensure that we cannot enter a wrong argument so this test is not useful anymore